### ISSUE TITLE
Advanced Item Search

### ISSUE THREAD

**Issue by LeGuybrush** [Takenbacon/wowarmory#8]
I assume this is a feature that has yet to be completed or is not available. At any rate, here's an issue submission for it.

```
/armory/item-search.xml has no XSLT. Expected at /armory/_layout/items/search.xsl
```

**Comment by Takenbacon** [comment]
Yeah, the layout doesn't exist, so we are pretty much screwed with this page unless Shadez or someone has the source of it from the real WoWArmory, which is very unlikely.

**Comment by Takenbacon** [comment]
I found this: http://web.archive.org/web/20090904131912/http://www.wowarmory.com/item-search.xml

The javascript doesn't seem to work, but the structure and html are all there. I may be able to do something with it in the future, but for now it'd just simply take too much time.
